js/error.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+(function () {
+  var template = document.querySelector('#error')
+    .content
+    .querySelector('div');
+
+  window.error = {
+    handler: function (errorMessage) {
+      var element = template.cloneNode(true);
+      element.querySelector('.error__message').textContent = errorMessage;
+      document.querySelector('main').appendChild(element);
+
+      document.addEventListener('keydown', errorKeydownHandler);
+      element.addEventListener('click', errorClickHandler);
+    }
+  };
+
+  var errorKeydownHandler = function (evt) {
+    window.submitButton(evt, closeXhrErrorWindow());
```

Comment:
```suggestion
    window.submitButton(evt, closeXhrErrorWindow);
```

этож ты колбек передаешь? ему же не надо сразу выполняться?